operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3278,14 +3278,17 @@ graphite:
     # -- Pod Disruption Budget
     podDisruptionBudget: {}
 
-    # -- SecurityContext override for graphite web pods
-    securityContext: {}
+    # -- SecurityContext override for graphite-web pods
+    # graohite-web pods need to run as root
+    # Because of that the graphite-web pods are incompatible with the PodSecurityPolicy of the
+    # mimir-distributed chart and with the Restricted policy of Pod Security Standards https://kubernetes.io/docs/concepts/security/pod-security-standards/
```

Comment:
Is there a way to circumvent this? Can graphite-web be disabled or is it critical for the graphite proxy?

Reply:
graphite-web can be disabled, considering removing it entirely from the Helm chart actually instead of giving it elevated permissions. Most queries don't need graphite-web (currently getting some stats on this), and I don't think graphite-web has actually ever run successfully in the Helm chart, even before the restricted security policy stuff, due to port/request host configs in the graphite-web image.

Reply:
I haven't heard of complaints since https://github.com/grafana/mimir/pull/3007 was merged, so maybe we can safely remove it.

But this also makes me realize that we don't test the graphite components in any way. We have some smoke tests that just start up the components from the chart with a few basic setups. The reason we don't test graphite is that we don't have a GEM license in CI. There is this issue which brings this up https://github.com/grafana/mimir/issues/2689

I can try to get a test license in CI and run graphite with it - that is after we've fixed graphite web